src/rgw/store/sfs/CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to
 
 - Ability to list buckets via admin REST API
 
```

Comment:
please, remove the blank line here

Reply:
what blank line? the one after ```Delete buckets.``` ? 
Or the blank lines separating items?

Reply:
line 17, for consistency with the rest of the file

Reply:
That was not part of the PR (was already there), but changed.